### ISSUE TITLE
Question about the criteria of transfering bad pokemons

### ISSUE THREAD

**Issue by zyf911** [finndev/PokeBuddy#36]
I just start using this software today. Quick question: when I check the box of "transfer bad Pokemon", what are the criteria? CP<a number or iv< certain %?

Thanks~

Yufan

**Comment by efraim14** [comment]
Yeah need to know this aswell. Having my bag full of 250/250 Pokemons but i have tons of pokemon i dont want.

**Comment by wolfwood69** [comment]
how about replacing transfering bad pokemons in pokebuddy with transfer criteria on pokecrot 2.7 gui (https://www.pokecrot.com/dl/), 
it have bot setting to 1). autotransfer double pokemon and max double pokemon size 
           2). max cp to transfer 
but pokecrot 2.7 gui doesn't work well to catch any pokemon

**Comment by YawYawz** [comment]
Please make options for this, I recommned Criteria to be IV, CP and Move Sets..
